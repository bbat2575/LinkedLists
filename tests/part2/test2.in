NEW 5
1
+
2
=
3
DELETE 0 3
DELETE 0 1
DELETE 0 2
DELETE 0 1
NEW 5
Once
upon
a
time
...
DELETE 1 4
DELETE 1 -3
DELETE 1 2
DELETE 1 1
DELETE 1 0
NEW 4
2e4
22.22
0.1
2e-4
DELETE 2 -1
DELETE 2 2
DELETE 2 -2
DELETE 2 0
NEW 7
99                                                                                                                             
                                                                                                                             99
9.9                                                                                                                            
                                                                                                                            9.9
$                                                                                                                              
                                                                                                                              $
Once upon a tiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiime
DELETE 3 -7
DELETE 3 -6
DELETE 3 0
DELETE 3 1
DELETE 3 -3
DELETE 3 0
DELETE 3 -1
