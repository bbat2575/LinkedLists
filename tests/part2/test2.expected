
Type HELP for a list of options
Press <CTRL + D> to Quit

> List 0: 1 -> + -> 2 -> = -> 3

> List 0: 1 -> + -> 2 -> 3

> List 0: 1 -> 2 -> 3

> List 0: 1 -> 2

> List 0: 1

> List 1: Once -> upon -> a -> time -> ...

> List 1: Once -> upon -> a -> time

> List 1: Once -> a -> time

> List 1: Once -> a

> List 1: Once

> List 1: 

> List 2: 20000.00 -> 22.22 -> 0.10 -> 0.00

> List 2: 20000.00 -> 22.22 -> 0.10

> List 2: 20000.00 -> 22.22

> List 2: 22.22

> List 2: 

> List 3: 99 -> 99 -> 9.90 -> 9.90 -> $                                                                                                                               ->                                                                                                                               $ -> Once upon a tiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiime

> List 3: 99 -> 9.90 -> 9.90 -> $                                                                                                                               ->                                                                                                                               $ -> Once upon a tiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiime

> List 3: 9.90 -> 9.90 -> $                                                                                                                               ->                                                                                                                               $ -> Once upon a tiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiime

> List 3: 9.90 -> $                                                                                                                               ->                                                                                                                               $ -> Once upon a tiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiime

> List 3: 9.90 ->                                                                                                                               $ -> Once upon a tiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiime

> List 3:                                                                                                                               $ -> Once upon a tiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiime

> List 3: Once upon a tiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiime

> List 3: 

> 
